data/json/items/gun/nail.json

```diff
@@ -1,5 +1,29 @@
 [
   {
+    "id": "coilgun",
+    "copy-from": "gun_base",
+    "type": "GUN",
+    "name": "coilgun",
+    "description": "A homemade gun, using electromagnets to accelerate a ferromagnetic projectile to high velocity.  Powered by UPS.",
+    "weight": 3341,
+    "volume": 11,
+    "price": 75000, "//": "Hard to make, plentiful and cheap ammo, and silent - people will want this thing.",
+    "bashing": 10,
+    "to_hit": -2,
+    "material": "steel",
+    "skill": "rifle",
+    "ammo": "nail",
+    "ranged_damage": 8,
```

Comment:
> sight dispersion was set specifically because it is not a factory made wrapon, so the sights are not going to be as good

We have craftable `improved iron sights` produced from a single piece of plastic and scrap metal at low skill levels that have better stats than factory weapons? This weapon is considerably more difficult to produce so why would the sights be poorer?

> buffing arbitrary stats

There is no IRL item to base this on. The description explicitly mentions higher velocity so that provides a ready explanation for increased `range`. Given the projectile is very light it presumably has similar `damage` upon striking a target.

That leaves us with `pierce`. Presently our implementation is limited to 1:1 reduction by armor so this increase simulates the idea that the faster projectile can pierce armor but not necessarily do much more `damage` when it does.